.github/workflows/build_conda_linux.yml

```diff
@@ -0,0 +1,46 @@
+name: Trigger Linux Conda Build for Nightlies/Release
```

Comment:
nit: can you rename this as "Build Linux Conda", and same below? See the job names we've used in text and vision as reference.

Reply:
This can be a follow-up PR, feel free to check this one in